playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: adhoc
+  pre_tasks:
+  - name: Print list of known health checks
+    action: openshift_health_check
+    when: checks is undefined or not checks
+  post_tasks:
+  - name: Run health checks
+    action: openshift_health_check
+    args:
+      checks: '{{ checks | default([]) }}'
```

Comment:
Yes, I agree. I think we went with `checks` for making it easier to type.

Still, I can't say I'm happy with such a long command:

```
$ ansible-playbook -i hosts playbooks/byo/openshift-checks/adhoc.yml -e checks=fluentd
```

In this case `oc diagnostics` (that could have a short alias like `oc diag`) is much easier on the eyes and less keystrokes. Surely keystrokes matter less for scripting.

If you think `openshift_checks` is more appropriate, we can plan a migration to that.